ArjunaJTS/narayana-jts-ibmorb/src/test/resources/ConfigurationInfo.properties

```diff
@@ -0,0 +1,3 @@
+sourceId=${buildNumber}
+propertiesFileName=${propertiesFileName}
+buildId=JBoss Inc. [${user.name}] ${os.name} ${os.version} ${buildproperty.date}
```

Comment:
Duplication. This file is not needed

Reply:
If we provide `testResource`, we would need this file in `test` directory, but I can also give the reference of file that is present in `src` directory, but that would not be a good approach I guess.

Reply:
I do not think you need a reference to a resource during tests